source/mainnet/net/installation/previous-node-downloads.rst

```diff
@@ -0,0 +1,198 @@
+
+:orphan:
+
+.. _previous-downloads:
+
+=======================
+Previous node downloads
+=======================
+
+This topic contains downloads for previous node versions if you need to upgrade a node over multiple versions.
+
+Windows - Mainnet and Testnet
+=============================
+
+**Please be aware that you should backup your configuration, as the installer will overwrite the current configuration with a standard configuration.**
+
+`Windows 4.2.2 <https://distribution.concordium.software/windows/Signed/Node-4.2.2-0.msi>`_
+
+`Windows 4.2.1 <https://distribution.concordium.software/windows/Signed/Node-4.2.1-0.msi>`_
+
+`Windows 4.1.1 <https://distribution.concordium.software/windows/Signed/Node-4.1.1.msi>`_ (Note: new genesis)
+
+`Windows 4.1.0 <https://distribution.concordium.software/windows/Signed/Node-4.1.0-0.msi>`_
+
+`Windows 4.0.11 <https://distribution.concordium.software/windows/Signed/Node-4.0.11-0.msi>`_
+
+`Windows 4.0.7 <https://distribution.concordium.software/windows/Signed/Node-4.0.7.msi>`_
+
+`Windows 3.0.2 <https://distribution.concordium.software/windows/Signed/Node-3.0.2.msi>`_
+
+`Windows 3.0.1 <https://distribution.concordium.software/windows/Signed/Node-3.0.1.msi>`_
```

Comment:
I think we should remove 3.0.1 and 3.0.0 since they are the same as 3.0.2 minus a security fix.